apps/avifenc.c

```diff
@@ -230,7 +230,7 @@ static void syntaxLong(void)
     printf("                                        Use 2 for any you wish to leave unspecified\n");
     printf("    -r,--range RANGE                  : YUV range, one of 'limited' or 'l', 'full' or 'f'. (JPEG/PNG only, default: full; For y4m or stdin, range is retained)\n");
     printf("    --target-size S                   : Set target file size in bytes (up to 7 times slower)\n");
-    printf("    --progressive                     : EXPERIMENTAL: Auto set parameters to encode a simple layered image supporting progressive rendering from a single input frame.\n");
+    printf("    --progressive                     : EXPERIMENTAL: Automatically set parameters to encode a simple layered image supporting progressive rendering from a single input frame.\n");
```

Comment:
'Automatically' does sound a bit redundant here.

Reply:
I looked into this. The reason "automatically" and "automatic" are used to describe the `--progressive` option is that the `--layered` option below (line 234) can also be used to encode a layered image but the parameters for each layer must be set manually.

Reply:
Could the `--progressive` doc describe what `--layered` arg it is equivalent to?